bin/picca_deltas.py

```diff
@@ -705,13 +761,16 @@ def main():
     deltas = {}
     data_bad_cont = []
     for healpix in sorted(data.keys()):
-        deltas[healpix] = [
-            Delta.from_forest(forest, get_stack_delta, Forest.get_var_lss,
-                              Forest.get_eta, Forest.get_fudge,
-                              args.use_mock_continuum)
-            for forest in data[healpix]
-            if forest.bad_cont is None
-        ]
+        for forest in data[healpix]:
```

Comment:
Why are you removing the list comprehension? How does this affect performance?

Reply:
Because we don't create Delta objects anymore. We modify them. 
Performance here is definitely not a bottleneck.

Reply:
makes sense